# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1368449945 14400
#      Mon May 13 08:59:05 2013 -0400
# Node ID cfacb0446066dbe382004dd0b4d0e8854fd1e57c
# Parent  4e50bd2946d8563d3e201cc04b3ba0720c991b06
Added tag ss-3-7-4 for changeset 4e50bd2946d8

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -71,8 +71,9 @@ a95432e7309ca6fc776c02939264bb6d443f3525
 2e8eb9ac43a5f8cfaf0423814a312ed47cb80485 rc-3-6-4-0
 df1aceb8f0bc6b5b5062907931cc663467f57d93 ss-3-7-1
 858cbf6fc2ec1c232f5cf1d75dc344439b39a89c rc-3-6-4-1
 faefa1bea8ddae3cab170afdeab68d3d15c4e623 ss-3-7-2
 0000000000000000000000000000000000000000 ss-3-7-2
 23a7661e529ae9bfc91693618f8c314c31f695ca ss-3-7-2
 cc5a7d1233f3acea85648baeb754fc0e8f225225 rc-3-6-4-2
 b29b10fbb7448cdfe29322446e1a589e7fe1a40a release-3-6-4
+4e50bd2946d8563d3e201cc04b3ba0720c991b06 ss-3-7-4
